src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(Color.values().length);
+        return Color.values()[randomNumber];
```

Comment:
[Use name() for getting String representation of enum constants](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)

Reply:
I don't understand what you mean, in this part of code method getRandomColor() returns type Color is not String.